index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7507e0edea.js" crossorigin="anonymous"></script>
+        <title>Personal portfolio</title>
+</head>
+<body>
+    <div>
+    <nav class="bar">
+       <a href="#"><h2 class="logo">My Logo</h2></a>
+        <div class="icon">
+            <a href="#" alt="bars"><i class="fa-solid fa-bars"></i></a>
+        </div>
+    </nav>
+</div>
+    <div>
+<header>
+    <h1 class="name">I’m JIPSON <br>
+        Glad to see you!</h1>
+  </header>
+        <p class="ABout"><b>I’m a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t hestiate to contact me.</b></p>
+        </div>
+            <br>
+                <h3>LET'S CONNECT</h3>
+            <div class="container"> 
+              <a href="#"><i class="fab fa-facebook-f"></i></a>
+              <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>  
+            </div>
+            <div class="rectangle4"></div>
+            <div class="elipse3"></div>
+            <div class="elipse15"></div>
+            <div class="elipse14"></div>
+            <div class="elipse8"></div>
+            <div class="elipse11"></div>
+            <div class="rectangle6"></div>
+            <div class="elipse12"></div>
+            <div class="elipse13"></div>
+            <div class="elipse4"></div>
+            <div class="rectangle5"></div>
+            <div class="elipse5"></div>
+            <div class="rectangle7"></div>
```

Comment:
- [ ] Kindly remove the background images from HTML and put them in CSS as `background-image` of the section. There is no need to download each shape on its own, you can download the whole picture directly from Figma. Kindly check the instructions in this [video](https://www.loom.com/embed/167236d17f104fc18298c5c9888354c9).